README.md

```diff
@@ -12,9 +12,23 @@ holds a list of 128-bit GATT UUIDs that
 websites using the Web Bluetooth API are forbidden from accessing.
 This includes all of Services, Characteristics, and Descriptors.
 
-This file contains comments in lines starting with `#`.
-Non-comment lines contain a [valid UUID](https://webbluetoothcg.github.io/web-bluetooth/#valid-uuid)
+[manufacturer_data_blocklist.txt](/manufacturer_data_blocklist.txt)
+holds a list of manufacturer data that websites using the Web Bluetooth API are forbidden from accessing.
+Each entry of the list consists of a company identifier and data prefix that should be blocked.
+
+These files contain comments in lines starting with `#`.
+Non-comment lines contain one of these:
+  * A [valid UUID](https://webbluetoothcg.github.io/web-bluetooth/#valid-uuid)
 followed optionally by a space and one of the tokens "`exclude-reads`" or "`exclude-writes`".
+  *  The word `manufacturer` followed by a space and a group of hexadecimal digits (at most four
+  hexadecimal digits) that represents a company identifier, which can be found on the
+  [Bluetooth Assigned Numbers website](https://www.bluetooth.com/specifications/assigned-numbers).
+  A space after that is an advertising data prefix in the format `advdata-<data>/<mask>`, where
+  `data` and `mask` are strings of hexadecimal digits of the same length. The byte order of the data
```

Comment:
small change please on wording, such as:


... strings of bytes in hexadecimal, e.g. `advdata-001100/00FF00`.

A data prefix matches an advertising data if each byte of the prefix is equal to the corresponding byte of the advertising data after both are ANDed with the corresponding byte of the mask.

E.g. the example earlier would match data such as 
`11` `11` `11`, or 
`00` `11` `22`, etc.